packages/eslint-plugin/docs/rules/no-empty-interface.md

```diff
@@ -38,6 +38,23 @@ interface Bar {
 interface Baz extends Foo, Bar {}
 ```
 
+### Options
+
+This rule accepts a single object option with the following default configuration:
+
+```json
+{
+  "@typescript-eslint/no-empty-interface": [
+    "error",
+    {
+      "allowSingleExtend": false
```

Comment:
Sorry to be that guy :D, but I found it jarring that it was `...Extend`, singular.

IMO it should either be `allowSingleExtends` or `allowSingleExtension`, probably the first one to better match the keyword and AST.

Reply:
Don't worry! Better we do the right thing before having a bad API afterwards. I had been thinking quite some time about the name and I wasn't 100% sure if that's a good one so I'm totally open for a change here. I will update the PR to use `allowSingleExtends`

Reply:
I've updated the PR. Please check again